### PR TITLE
Add !tab to the commands

### PR DESCRIPTION
**Pull request by xpple**
Oftenly asked question, therefore useful to have a simple command for it

### CODE REVIEW

**Comment by Laarryy** [comment]
I am confused why you worked to do this 😂 
Typically it's a good idea to check already open PRs - I made one before yours 😄

File: modules/commands/list.json [hunk 1]
```diff
@@ -520,6 +520,24 @@
...
+    ],
+    "title": "Tablist",
+    "description": "LuckPerms is only able to send information to formatting plugins so that they display them properly. LuckPerms does not put the prefix in front of your name on the tablist, you need a tab formatting plugin to do it, and you need Vault. Same goes for the chat! If the information appears correctly when you run `/lp user <user> info`, LuckPerms is doing its job!",
+    "fields": [
+      {
+        "key": "Prefix isn't showing?",
+        "value": "https://github.com/lucko/LuckPerms/wiki/FAQ#why-are-prefixessuffixes-not-working"
+      },
+      {
+        "key": "How can I use TAB to format my tablist?",
```
Comment (Laarryy) [inline modules/commands/list.json:536]: Assuming that a person will use TAB is probably not a great idea for this embed when there are alternatives that should be suggested

File: modules/commands/list.json [hunk 2]
```diff
@@ -520,6 +520,24 @@
       }
     ]
   },
+  {
+    "name": "tab",
+    "aliases": [
+      "tablist"
+    ],
+    "title": "Tablist",
+    "description": "LuckPerms is only able to send information to formatting plugins so that they display them properly. LuckPerms does not put the prefix in front of your name on the tablist, you need a tab formatting plugin to do it, and you need Vault. Same goes for the chat! If the information appears correctly when you run `/lp user <user> info`, LuckPerms is doing its job!",
```
Comment (Laarryy) [inline modules/commands/list.json:529]: It is not necessarily the case that you need Vault to display a prefix in tab, as many tab plugins simply use PAPI

**Review by Laarryy — COMMENTED** [review]
Further considerations should be made before submitting a PR which is just duplicating the code, probably 😛

**Comment by lucko** [comment]
Thanks, I've applied a (slightly modified) version of this PR here: 58d44882d82747a8bf3a88ab6fbc7e2a1fe9e437

**Pull request closed without merging**